app/views/spotlight/sir_trevor/blocks/_iframe_block.html.erb

```diff
@@ -0,0 +1 @@
+<%= sanitize iframe_block.code, tags: %w(iframe), attributes: %w(id width height src frameborder allowfullscreen sandbox seamless srcdoc) %>
```

Comment:
Other possible attributes we may want: `marginwidth`, `marginheight`, and `scrolling`.  There are probably more, but those will be a good start.

I know marginheight and marginwidth are not HTML5, but I think it's still used from time-to-time.